scripts/ci/gitlab/pipeline/build.yml

```diff
@@ -79,10 +79,12 @@ build-malus:
     - .compiler-info
```

Comment:
If a pre-this-PR code is built (without split-out binaries), `mv` would fail. I'm not sure if it makes the whole job fail but added this just to be explicit: "move the files if they are there, if not, we don't care".

Reply:
Got it, so it can be removed now?